README.md

```diff
@@ -71,4 +71,4 @@ We are hoping that it will support more countries; you can [help us](https://git
 
 ## Tile Server
 
-Americana's GitHub map uses a privately-donated community [tile server](https://tile.ourmap.us). It is available for use by other hobbyist and community projects. See also the [tile server usage policy](TILE_USAGE.md).
+While the source code in this repository remains dedicated to the public domain under a CC0 waiver, Americana's GitHub map uses a privately-donated community [tile server](https://tile.ourmap.us) to run the live map. This server is available for use by other hobbyist and community projects with constraints. See the [tile server usage policy](TILE_USAGE.md) for more details. This usage policy describes the allowable usage of the tiles for other projects, separate from this style or the application being developed in this repository.
```

Comment:
The use of this tile server is actually just a default:

```suggestion
Although the source code in this repository is dedicated to the public domain under a CC0 waiver, it is configured by default to load map tiles from a privately-donated community [tile server](https://tile.ourmap.us). This server is available for use by other hobbyist and community projects with constraints. See the [tile server usage policy](TILE_USAGE.md) for more details. This usage policy describes the allowable usage of the tiles for other projects, separate from this style or the application being developed in this repository.
```

Consider adding something here in the configuration guide alongside the section about MapTiler:

https://github.com/ZeLonewolf/openstreetmap-americana/blob/aba811e5aff7b075ba06d8bab19fe81657a0aea2/CONTRIBUTING.md#L104-L109